src/global/app-css.scss

```diff
@@ -67,3 +78,5 @@ body {
   color: var(--bloom-color-on-surface);
   line-height: inherit;
 }
+
+@import "text.scss";
```

Comment:
Is there a reason this import is on the bottom rather than the ones at the top?

Reply:
It's just because it's a stylesheet with classes rather than design tokens, and down the road we might have a few more stylesheets like that so they'd also import at the bottom.